cmake/Install.cmake

```diff
@@ -472,6 +457,29 @@ if( UNIX AND NOT APPLE )
   install(PROGRAMS "${CMAKE_BINARY_DIR}/scripts/runenergyplus" DESTINATION "./")
   install(PROGRAMS scripts/runepmacro DESTINATION "./")
   install(PROGRAMS scripts/runreadvars DESTINATION "./")
+
+  # You need at least one "install(..." command for it to be registered as a component
+  install(CODE "MESSAGE(\"Creating symlinks.\")" COMPONENT Symlinks)
+elseif(WIN32)
+
+  # Will also set CPACK_IFW_PACKAGE_START_MENU_DIRECTORY (name of default program group in Windows start menu)
+  set(CPACK_IFW_PACKAGE_NAME "EnergyPlusV${CPACK_PACKAGE_VERSION_MAJOR}-${CPACK_PACKAGE_VERSION_MINOR}-${CPACK_PACKAGE_VERSION_PATCH}")
+
+  set(CPACK_PACKAGE_INSTALL_DIRECTORY "${CPACK_IFW_PACKAGE_NAME}" )
+  set(CPACK_IFW_TARGET_DIRECTORY "C:/${CPACK_PACKAGE_INSTALL_DIRECTORY}" )
+
+    # Custom installer icon. Has to be .icns on mac, .ico on windows, not supported on Unix
+  set(CPACK_IFW_PACKAGE_ICON "${CMAKE_SOURCE_DIR}/release/ep.ico")
+
+  # You need at least one "install(..." command for it to be registered as a component
+  install(CODE "MESSAGE(\"Registering filetypes.\")" COMPONENT RegisterFileType)
+  install(CODE "MESSAGE(\"Copying and Registering DLLs\")" COMPONENT CopyAndRegisterSystemDLLs)
+  install(CODE "MESSAGE(\"Creating start menu.\")" COMPONENT CreateStartMenu)
+
+endif()
+
+if(UNIX)
+  install(FILES doc/man/energyplus.1 DESTINATION "./")
```

Comment:
Can this just go right above in the UNIX section with the IFW stuff?

Reply:
I moved this into the UNIX section above.  Note that when I ran the QtIFW-built installer on Linux, then ran the maintenance tool to remove it, it gave a warning while trying to delete the man file.  I was able to just click ignore and it worked fine otherwise, so no big deal.